sklearn/covariance/outlier_detection.py

```diff
@@ -28,27 +29,40 @@ class OutlierDetectionMixin(object):
         The amount of contamination of the data set, i.e. the proportion
         of outliers in the data set.
 
+    raw_decision : bool
+        Whether or not to consider raw negated Mahalanobis distances as the
+        decision function. Otherwise returns `[-dist**(1/3) + thres**(1/3)]`.
+        Must be False` (default) for compatibility with the others outlier
```

Comment:
That text was added before my time, but I did my best to interpret what was meant 😄 .